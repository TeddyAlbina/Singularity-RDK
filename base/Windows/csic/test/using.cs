using System;
using point = System.Drawing.Point;

public class test {
	public static void Main() {	(new test()).print(); }
	test() {
		for (int i = 0; i <= x.GetUpperBound(0); i++)
			for (int j = 0; j < x.GetUpperBound(1); j++)
				x[i,j] = new point(i, j);
	}
	void print() {
		foreach (point p in x)
			Console.WriteLine(p);
	}
	point[,] x = new point[4,4];
}
