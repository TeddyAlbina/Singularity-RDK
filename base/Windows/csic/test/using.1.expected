{X=0,Y=0}
{X=0,Y=1}
{X=0,Y=2}
{X=0,Y=0}
{X=1,Y=0}
{X=1,Y=1}
{X=1,Y=2}
{X=0,Y=0}
{X=2,Y=0}
{X=2,Y=1}
{X=2,Y=2}
{X=0,Y=0}
{X=3,Y=0}
{X=3,Y=1}
{X=3,Y=2}
{X=0,Y=0}
